components/contribution-flow/StepPayment.js

```diff
@@ -38,6 +38,7 @@ const StepPayment = ({
         stepSummary={stepSummary}
         stepDetails={stepDetails}
         stepPayment={stepPayment}
+        stepProfile={stepProfile}
```

Comment:
`stepProfile` is already passed as `fromAccount` prop. "stepProfile" is a better name, but we should consolidate